torch/ao/quantization/_pt2e/graph_utils.py

```diff
@@ -28,8 +28,21 @@ def _create_equivalent_types_dict():
     return _DICT
 
 
-_EQUIVALET_TYPES_DICT = _create_equivalent_types_dict()
-
+_EQUIVALENT_TYPES_DICT = _create_equivalent_types_dict()
+
+def get_equivalent_types() -> List[Set]:
+    return _EQUIVALENT_TYPES
+
+def update_equivalent_types_dict(customized_equivalent_types=None):
+    """Help function for user who wants to customize the _EQUIVALENT_TYPES and _EQUIVALENT_TYPES_DICT.
+    When customized_equivalent_types passes in,
+    re-generate _EQUIVALENT_TYPES and _EQUIVALENT_TYPES_DICT.
+    """
+    if customized_equivalent_types is not None:
```

Comment:
I would suggest you raise ValueError is update_equivalent_types==None. Because this function is explicitly for updating equivalent types

Reply:
Thanks for the suggestions and changed.